src/constants/license.js

```diff
@@ -1,22 +1,45 @@
-export const CC_LICENSES = [
+export const CC_LICENSES = /** @type {const} */ ([
```

Comment:
@sarayourfriend , can you give me a reference I can read about the `type {const}` declaration? I couldn't find anything searching for `typescript const type`.

The reason I'm asking this is the messages I see in my IDEs.
**WebStorm** is trying to find something in the `node_modules`, I guess?

<img width="1024" alt="Screen Shot 2022-02-15 at 3 14 24 PM" src="https://user-images.githubusercontent.com/15233243/154061395-d36a4ac2-af69-413f-a46c-addb64ec65e8.png">

**VS Code** just shows it as unresolved, even when I set language mode to TypeScript (I don't know what other information can I get about VS Code TS hints)

<img width="538" alt="Screen Shot 2022-02-15 at 3 14 45 PM" src="https://user-images.githubusercontent.com/15233243/154061463-f2031496-5552-44ee-a906-2dade09c067a.png">

Reply:
It's the JSDoc version of this feature: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

The TypeScript JSDoc documentation for it is here: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#casts (the second example in that section)

I'm not sure why TS isn't picking it up. In VSCode have you tried running the "Restart TS Service" command? I have to do that anytime `tsconfig.json` changes.

For WebStorm it might be the same. You will probably also have to enable the TS language server using the instructions described here: https://www.jetbrains.com/help/webstorm/typescript-support.html#ws_ts_use_ts_service_checkbox